personal/privacyidea/class_mfaAccount.inc

```diff
@@ -192,6 +211,18 @@ class mfaAccount extends plugin
         /* Call parent execute */
         plugin::execute();
 
+        // The apply action is handled automatically if we are a child of a
+        // management object, otherwise saving changed settings has to be
+        // handled manually
+        if (array_key_exists('edit_apply', $_POST)) {
+            $management = $this->getParentIsA("management");
+            if ($management === null) {
+                $this->save_object();
+                unset($_POST['edit_apply']);
+                return $this->execute();
+            }
+        }
+
```

Comment:
Maybe move this to main.inc? I have a suspicion that we don't need this in upstream GOsa².

@gber: if you use our Debian Edu GOsa² and go to the personal info page of a user, can you edit and apply settings there? If yes, then this bit should got into main.inc.

Reply:
(where main.inc is in the customer's plugin variant, not in this)

Reply:
It turns out that user self-management in Debian Edu GOsa² also embeds the plugin into `MyAccountTabs` which is derived from `tabs`.